### PR TITLE
Issue 16 adds nestedFolders command line option to generated Insomnia…

### PR DESCRIPTION
**Pull request by mag382**
… and Postman collections maintaining the folder structure of the original spring rest docs.

### CODE REVIEW

File: src/curl-to-insomnia3.js [hunk 1]
```diff
@@ -67,19 +66,82 @@ const addFolders = (determineFolder, resourceWrappers) => {
...
 
+/**
+ * Loop through the parts of a folder name delimited by "/", and create a folder in the appropriate location in the tree
+ * of folders for each part.
+ * @param nestedFolderPath
+ * @param folderNameToId
+ * @param folderResources
+ * @param resourceWrapper
+ * @param folderCount
+ * @returns {*}
```
Comment (fbenz) [inline src/curl-to-insomnia3.js]: `@param` and `@returns` should be removed if no documentation is added. Same below for `createFolder`.

**Review by fbenz — APPROVED** [review]
Thanks a lot for the contributions. I suggest to not list parameters in the documentation if they are not documented.

**Comment by mag382** [comment]
I started working on another feature enhancement for the replacements logic to handle replacing parts of the path in the urls.  While looking at this, I see there is a problem in this branch here.  The replacements of postman items won't work below the top level.  I'll work on a fix for that.

**Comment by fbenz** [comment]
```
Test Suites: 2 passed, 2 total
Tests:       21 passed, 21 total
Snapshots:   0 total
Time:        1.908s, estimated 2s
Ran all test suites.
```

**Pull request merged**